### PR TITLE
Update pointer specification

### PR DESCRIPTION
**Pull request by ansigroup**
Add missing info for pointer description:
`If the pointer is greater than the amount of output, runestone is a cenotaph.`

### CODE REVIEW

File: docs/src/runes/specification.md [hunk 1]
```diff
@@ -321,7 +321,8 @@ transaction.
 
 The `Pointer` field contains the index of the output to which runes unallocated
 by edicts should be transferred. If the `Pointer` field is absent, unallocated
-runes are transferred to the first non-`OP_RETURN` output.
+runes are transferred to the first non-`OP_RETURN` output. 
+If the pointer is greater than the amount of outputs, runestone is a cenotaph.
```
Comment (raphjaph) [inline docs/src/runes/specification.md:325]:
```suggestion
runes are transferred to the first non-`OP_RETURN` output. If the pointer is greater than the 
number of outputs, the runestone is a cenotaph.
```

**Review by raphjaph — CHANGES_REQUESTED** [review]
Thanks! Just a small change

**Review by raphjaph — APPROVED** [review]

**Comment by raphjaph** [comment]
I'm unable to update the branch, I think you disabled allow edits by maintainers

**Comment by ansigroup** [comment]
> I'm unable to update the branch, I think you disabled allow edits by maintainers

Interesting, I did not disable allow edits, and do not see this option here to enable it. But, It was saying that branch is already outdated and require new merge with master, what I just did.

**Comment by raphjaph** [comment]
Ok cool, will merge now!

**Comment by ansigroup** [comment]
Now I see it again out-of-date with the base branch... 

image

**Pull request merged**